apis/dataprotection/v1alpha1/types.go

```diff
@@ -41,6 +41,16 @@ const (
 	BackupTypeSnapshot BackupType = "snapshot"
 )
 
+// BackupMethod the backup method
+// +enum
+// +kubebuilder:validation:Enum={snapshot,backupTool}
```

Comment:
missing `none and all`

Reply:
Now, `backupMethod` only specify the backup method, include two methods, `snapshot` or use `backupTool`. The `enabled` field specify if enable the `datafile` backup, this field specify how to execute the `datafile` backup.